### PR TITLE
Fixed PEP8 linewidth and non-public naming conventions. Fixed pathing issues for test cases.

### PR DESCRIPTION
**Pull request by monshap**

### CODE REVIEW

File: plepy/PLEpy.py [hunk 1]
```diff
@@ -13,13 +13,13 @@ def __init__(self, model, pnames: list, indices=None, solver='ipopt',
                  solver_kwds={}, tee=False, dae=None, dae_kwds={},
                  presolve=False):
         """Profile Likelihood Estimator object
-        
+
```
Comment (paulboone) [inline plepy/PLEpy.py:16]: white space should be in a separate commit

File: plepy/PLEpy.py [hunk 2]
```diff
@@ -85,6 +85,16 @@ def __init__(self, model, pnames: list, indices=None, solver='ipopt',
...
+        """Indicate which index to use for parameter, pname
+
+        Args
+        ----
+        pname : str
+            name of parameter to set index of
+        *args : str
+            string specifying the name of the index or indices to use
+            for parameter
+        """
```
Comment (paulboone) [inline plepy/PLEpy.py:97]: this should be a separate commit

File: plepy/PLEpy.py [hunk 3]
```diff
@@ -101,20 +111,20 @@ def set_index(self, pname: str, *args):
             # get parameter bounds
             self.pbounds[pname][k] = self.plist[pname][k].bounds
 
-    def getval(self, pname: str):
+    def _getval(self, pname: str):
```
Comment (paulboone) [inline plepy/PLEpy.py:114]: this shouldn't be necessary to PEP8, but is ok if you want to do it this way.

File: tests/fiveshell/shell_example.py [hunk 4]
```diff
@@ -3,13 +3,17 @@
 import matplotlib.pyplot as plt
 import seaborn as sns
 import pandas as pd
+import os
 import sys
-sys.path.append("../")
+sys.path.append(os.path.abspath("../../"))
```
Comment (paulboone) [inline tests/fiveshell/shell_example.py:8]: none of this would be necessary if the package was setup correctly.

**Review by paulboone — COMMENTED** [review]
Decide how you want to do your method names!

File: tests/fiveshell/shell_example.py [hunk 5]
```diff
@@ -1,15 +1,19 @@
-# %% Problem Set-up
+# Problem Set-up
 import numpy as np
 import matplotlib.pyplot as plt
 import seaborn as sns
 import pandas as pd
+import os
```
Comment (paulboone) [inline tests/fiveshell/shell_example.py:6]: order of imports should always be:
python modules
modules installed via pip
modules from this package

**Comment by monshap** [comment]
Changed non-public method names back (no leading underscore), updated the requirements, and fixed the imports (order and wildcards).

**Comment by paulboone** [comment]
The new commits look good, but you do need to rebase!

**Comment by monshap** [comment]
I think this should be ready to merge unless you see anything else.

File: README.md [hunk 6]
```diff
@@ -30,7 +30,7 @@ python examples/rapidTEG/rapidTEG_example.py
 ```
 
 ## Quick Start
-There are 6 key steps for using PLEpy:  
+There are 6 key steps for using PLEpy:
```
Comment (paulboone) [inline README.md:33]: this can be squashed with your other whitespace commit

File: tests/fiveshell/shell_example.py (hunk 4 above)
Comment (paulboone) [reply tests/fiveshell/shell_example.py]: I am tempted to reject this commit because you definitely don't want to do this. But since you were already doing a form of this, we're going to leave it in place for now. But we definitely need to revisit in a couple weeks--after implementing click and getting your package set up properly--and make sure we delete all the appending to the sys.path!

**Review by paulboone — APPROVED** [review]
Looks great! The sys.path stuff will give you trouble, but we will figure that out after you get click working. You could have squashed your second whitespace commit in with your first, but that is super minor, so I'm merging!
